client/packages/system/src/Item/Components/ListItems/ListItems.tsx

```diff
@@ -50,21 +52,39 @@ export const ListItems = ({
     options.push({ id: 'new', value: t('label.new-item') });
   }
 
-  return (
-    <Tooltip title={value?.name}>
-      <Box sx={{ flexGrowY: 1, overflow: 'auto', scrollBehavior: 'smooth' }}>
-        <ListOptions
-          currentId={value?.id ?? 'new'}
-          onClick={id => {
-            if (currentItemId === 'new' && isDirty) {
-              showSaveConfirmation();
-            } else navigate(route.addPart(id).build());
-          }}
-          options={options}
-          enteredLineIds={enteredLineIds}
-          scrollRef={scrollRef}
-        />
-      </Box>
-    </Tooltip>
-  );
+  if (tooltipDisabled === true) {
+    return (
+        <Box sx={{ flexGrowY: 1, overflow: 'auto', scrollBehavior: 'smooth' }}>
+          <ListOptions
+            currentId={value?.id ?? 'new'}
+            onClick={id => {
+              if (currentItemId === 'new' && isDirty) {
+                showSaveConfirmation();
+              } else navigate(route.addPart(id).build());
+            }}
+            options={options}
+            enteredLineIds={enteredLineIds}
+            scrollRef={scrollRef}
+          />
+        </Box>
+    );
+  } else {
+    return (
+      <Tooltip title={value?.name}>
+        <Box sx={{ flexGrowY: 1, overflow: 'auto', scrollBehavior: 'smooth' }}>
+          <ListOptions
+            currentId={value?.id ?? 'new'}
+            onClick={id => {
+              if (currentItemId === 'new' && isDirty) {
+                showSaveConfirmation();
+              } else navigate(route.addPart(id).build());
+            }}
+            options={options}
+            enteredLineIds={enteredLineIds}
+            scrollRef={scrollRef}
+          />
+        </Box>
+      </Tooltip>
+    )
+  };
```

Comment:
This is on the right path!

Tooltips doesn't show if the [title](https://arc.net/l/quote/mmznnztp) prop is either `null`, `undefined`, `false` , or string have zero length.

We should be able to reduce the code through changing `title` only to `null` (or whatever value is preferred) if `tooltipDisabled` is `true`.